.circleci/config.yml

```diff
@@ -48,6 +49,9 @@ jobs:
             - "1d:f2:37:1e:7e:38:02:e0:76:2d:6a:a8:47:2e:85:09"
       - checkout
       - ruby/install-deps
+      - aws-cli/setup:
```

Comment:
add `session-duration` with a lower value?

Reply:
nice catch, will changae the duration to 2mins